README.md

```diff
@@ -30,12 +30,16 @@ Thumbnailing:
 
   # Resize to fill
   open("input.jpg") |> resize_to_fill("450x300") |> save
+  
+  # Resize to fill with gravity
+  open("input.jpg") |> resize_to_fill("450x300", gravity: "Center") |> save
```

Comment:
I think I'd expect, for consistency with the rest of the API:

```
open("input.jpg") |> resize_to_fill("450x300") |> gravity("Center") |> save
```

So the extra `resize_to_fill/3` is kind of redundant.